test/jdk/tools/launcher/I18NArgTest.java

```diff
@@ -125,5 +133,20 @@ static void testCharacters(String... args) {
                 "expected:" + expected + " obtained:" + hexValue;
             throw new RuntimeException(message);
         }
+        if (args.length == 3) {
+            // verify the value of the system property matches the expected value
+            String sysPropName = args[2];
+            String sysPropVal = System.getProperty(sysPropName);
+            if (sysPropVal == null) {
+                throw new RuntimeException("Missing system property " + sysPropName);
+            }
+            var sysPropHexVal = HexFormat.of().formatHex(sysPropVal.getBytes(StandardCharsets.UTF_16));
+            System.out.println("System property " + sysPropName + " computed hex value: "
+                    + sysPropHexVal);
+            if (!sysPropHexVal.contains(expected)) {
+                throw new RuntimeException("Unexpected value in system property, expected "
+                        + expected + ", but got " + sysPropHexVal);
+            }
+        }
     }
-}
+}
```

Comment:
Nit: don't remove the newline at the end

Reply:
I am sorry for my mistake. I fixed it.